pom.xml

```diff
@@ -256,87 +254,6 @@
         </plugins>
       </build>
     </profile>
-
```

Comment:
I'm guessing this is coming from java-shared-config?

Reply:
Yes exactly.

Reply:
Updated the commit message to make this clear.